src/CalcManager/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_library(CEngine
+	CEngine/calc.cpp
+	CEngine/CalcInput.cpp
+	CEngine/CalcUtils.cpp
+	CEngine/History.cpp
+	CEngine/Number.cpp
+	CEngine/Rational.cpp
+	CEngine/RationalMath.cpp
+	CEngine/scicomm.cpp
+	CEngine/scidisp.cpp
+	CEngine/scifunc.cpp
+	CEngine/scioper.cpp
+	CEngine/sciset.cpp
```

Comment:
How is the resulting lib functional if you haven't compiled Ratpak?  There is no definition for the various extern functions that are compiled into this library through the included ratpak headers.  Is that intentional?  Are you wanting to add support in pieces?  Each update that we merge into master should be functional so it seems like this file needs more work.

Reply:
The resulting library was indeed not functional without Ratpack, which should be included now though. (Ratpack support was previously tracked by https://github.com/fwcd/WindowsCalculator/pull/2)